pylintrc

```diff
@@ -96,7 +93,7 @@ ignore-imports = no
 [TYPECHECK]
 ignore-mixin-members = yes
 ignored-classes = SQLObject
-zope = no
+unsafe-load-any-extension = yes
```

Comment:
This came through with the new version of edx-lint. [Here's the reference.](https://github.com/edx/edx-lint/blob/faba6920bcbbc91a1064d3071723e095197b190f/edx_lint/files/pylintrc#L199)